code/datums/status_effects/debuffs/staggered.dm

```diff
@@ -6,13 +6,12 @@
 
 /datum/status_effect/staggered/on_creation(mob/living/new_owner, duration = 10 SECONDS)
 	src.duration = duration
+	if(new_owner.stat == DEAD)
+		return FALSE
```

Comment:
```suggestion
```
Add a condition to the `/tick` proc so that the mob doesn't do the stagger animation when dead instead.